python/cuspatial/cuspatial/core/gis.py

```diff
@@ -25,10 +25,41 @@ def directed_hausdorff_distance(x, y, count):
     ----------
```

Comment:
I can't comment on it, but I don't think the top definition of what this function does is correct (or at least it's inconsistent with the C++ function docs). 

Should be "Compute the directed Hausdorff distances between all pairs of input trajectories" (not "any groupings of polygons").

Reply:
Hausdorff doesn't explicitly compute over "trajectories", but I'm a fan of consistency! :thumbsup:

Reply:
Maybe not in general, but the input is in the form of what cuSpatial calls trajectories, is it not? I guess in general they are "coordinate lists", but I think the expectation is that they will be created using `derive_trajectories`?